src/Pages/Main/Slider.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect, useRef } from "react";
+import Slide from "./Slide";
+import styled from "styled-components";
+
+const TOTAL = 3;
+export default function Slider() {
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const slideRef = useRef(null);
+
+  const nextSlide = () => {
+    if (currentSlide >= TOTAL) {
+      setCurrentSlide(0);
+    } else {
+      setCurrentSlide(currentSlide + 1);
+    }
+  };
+
+  const prevSlide = () => {
+    if (currentSlide === 0) {
+      setCurrentSlide(TOTAL);
+    } else {
+      setCurrentSlide(currentSlide - 1);
+    }
+  };
+
+  useEffect(() => {
+    slideRef.current.style.transition = "all 0.5s ease-in-out";
+    slideRef.current.style.transform = `translateX(-${currentSlide * 100}%)`;
+  }, [currentSlide]);
+
+  return (
+    <Container>
+      {currentSlide}
+      <SliderContainer ref={slideRef}>
+        <video muted loop autoPlay>
+          <source
+            src="https://www.laneige.com/kr/ko/layout/main/key-visual/__icsFiles/afieldfile/2020/04/14/Laneige_WBE_Main_PC.mp4"
+            type="video/mp4"
+          />
+        </video>
+        <video muted loop autoPlay>
+          <source
+            src="https://www.laneige.com/kr/ko/layout/main/key-visual/__icsFiles/afieldfile/2020/09/01/perfect-renew-youth-regenerator-mainkv_pc_2_1.mp4"
+            type="video/mp4"
+          />
+        </video>
+        <Slide
+          img={`https://www.laneige.com/kr/ko/layout/main/key-visual/__icsFiles/afieldfile/2020/08/18/20200812_water-sleeping-mask-limited-edition_KV_pc_01_1.png`}
+        />
+        <Slide
+          img={`https://www.laneige.com/kr/ko/layout/main/key-visual/__icsFiles/afieldfile/2020/06/17/20200617_neo-cushion_KV_pc_1.png`}
+        />
+      </SliderContainer>
+      <ButtonContainer>
+        <Button onClick={prevSlide}>Previous</Button>
+        <Button onClick={nextSlide}>Next</Button>
+      </ButtonContainer>
+    </Container>
+  );
+}
+
+const Container = styled.div`
+  position: relative;
+  width: 100%;
+  overflow: hidden;
+`;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  margin: 0 auto;
+  position: absolute;
+  bottom: 2px;
+`;
+
+const Button = styled.button`
+  border: 1px solid black;
+`;
+
+const SliderContainer = styled.div`
+  width: 100vw;
+  height: 820px;
+  display: flex;
+
+  video {
+    width: 100 !important;
```

Comment:
```suggestion
    width: 100px;
```
- 단위가 없습니다
- important 꼭 필요한가요?